pushjack/apns.py

```diff
@@ -129,7 +129,6 @@ def close(self):
 
     def send(self,
              ids,
-             message,
```

Comment:
I think this should just be `message=None` instead of being removed completely. A `None` value is something that is already supported as mentioned in the docstring parameter description. Removing `message` would be a pretty big backwards compatibility issue.